testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1674,3 +1693,39 @@
   node = get_target(target)
   node.run("iptables -F INPUT")
 end
+
+When(/^I generate the configuration "([^"]*)" of Containerized Proxy on the server$/) do |file_path|
+  # Doc: https://www.uyuni-project.org/uyuni-docs/en/uyuni/reference/spacecmd/proxy_container.html
+  command = "echo spacewalk > cert_pass && spacecmd -u admin -p admin proxy_container_config_generate_cert" \
+            " -- -o #{file_path} -p 8022 #{$proxy.full_hostname.sub('pxy', 'pod-pxy')} #{$server.full_hostname}" \
+            " 2048 galaxy-noise@suse.de --ca-pass cert_pass" \
+            " && rm cert_pass"
+  $server.run(command)
+end
+
+When(/^I add avahi hosts in Containerized Proxy configuration$/) do
+  if $server.full_hostname.include? 'tf.local'
+    hosts_list = ""
+    $host_by_node.each do |node, _host|
+      hosts_list += "--add-host=#{node.full_hostname}:#{node.public_ip} "
+    end
+    hosts_list = escape_regex(hosts_list)
+    regex = "s/^#?EXTRA_POD_ARGS=.*$/EXTRA_POD_ARGS=#{hosts_list}/g;"
+    $proxy.run("sed -i.bak -Ee '#{regex}' /etc/sysconfig/uyuni-proxy-systemd-services")
+    log "Avahi hosts added: #{hosts_list}"
+    log 'Development team has not been working to support Avahi in Containerized Proxy. This is best effort.'
+  else
+    log 'Record not added - avahi domain is not detected'
```

Comment:
```suggestion
    log 'Record not added - avahi domain not detected.'
```
Or `was not detected`